students/PeterB/Session 2/Grid.py

```diff
@@ -0,0 +1,32 @@
+plus = '+'
```

Comment:
you really don't want a space in filenames -- it makes using the command line challenging. you can change it in git like so:

git mv "Session 2" Session2

you need to quotes so that git doesn't interpret the space as new name.